src/test/java/io/outfoxx/swiftpoet/test/FileSpecTests.kt

```diff
@@ -316,9 +330,12 @@ class FileSpecTests {
             import Foundation
             import some_module
             import some_other_module
+            import yet_another_module
 
             struct SomeType {
 
+              var yet_another_module_order: [yet_another_module.SortOrder]
```

Comment:
This test fails without my change. It outputs `var yet_another_module_order: [SortOrder]` which conflicts with the same module `SortOrder` that is added here.